mobile/src/reducers/calendar/calendar-events.reducer.ts

```diff
@@ -19,9 +20,14 @@ export interface SelectionSubState {
     selection: CalendarSelection;
 }
 
+export interface EventsMapSubState {
+    events: Map<string, CalendarEvent[]>;
```

Comment:
wrong kind of Map. You use JS built-in Map, but you should use one from Immutable library. Later you have `events.set`,  where you mutate it. That's bad because your components won't know that map has changed.

Reply:
ok